astropy/visualization/wcsaxes/coordinate_helpers.py

```diff
@@ -207,6 +207,16 @@ def set_coord_type(self, coord_type, coord_wrap=None):
 
         self.coord_type = coord_type
 
+        if coord_wrap is not None:
+            if not isinstance(coord_wrap, u.Quantity):
+                # import pdb; pdb.set_trace()
+                warnings.warn(
+                    "Passing 'coord_wrap' as a number is deprecated. Use a Quantity with units convertible to angular degrees instead.",
+                    AstropyDeprecationWarning,
+                )
+            else:
+                coord_wrap = coord_wrap.to_value(u.deg)
+
         if coord_type == "longitude" and coord_wrap is None:
             self.coord_wrap = 360
```

Comment:
Does this have to be Quantity too?

Reply:
No, from line 218 `coord_wrap` is a bare number.

Reply:
Isn't it going to be confusing if the (public) property is a plain number but a Quantity was passed in? I assume not many people would use that property so maybe it is safe to change it to a Quantity?

Reply:
Ahh good shout. I've updated this so `coord_wrap` is always stored as a number now.